Integrations/integration-cisco-meraki.yml

```diff
@@ -102,28 +103,39 @@ script:
         return text;
     }
 
+    var merakiUrlPrefix = 'https://dashboard.meraki.com/api/v0/';
+    var apiKey = params.apiKey;
+
     var sendRequest = function(url, method, body) {
         var requestUrl = merakiUrlPrefix + url;
-        var apiKey = params.apiKey;
         var requestMethod = method || 'GET';
 
-
         var httpParams = {
             Method: requestMethod,
```

Comment:
kine 111 should be here, no other usage of method. probably the same for line 110